src/lib/tvgRender.h

```diff
@@ -23,6 +23,7 @@
 #define _TVG_RENDER_H_
 
 #include "tvgCommon.h"
+#include <vector>
```

Comment:
Please include it in the actual necessary file.

Reply:
The vector is being used at line 68.